### ISSUE TITLE
[20.03.2020] Tiến độ tiến hành chương Linear Neural Networks

### ISSUE THREAD

**Issue by duythanhvn** [aivivn/d2l-vn#671]
Hy vọng chương **Linear Neural Networks** này gói gọn trong tháng 3/2020. Mình tạm để mốc thời hạn là **20.03.2020**.

## Giai đoạn 1 `(59 parts)`
* [x] 3. Linear Regression Index `01/01 part`
* [x] 3.1. Linear Regression `18/18 parts`
* [x] 3.2. Linear Regression Implementation from Scratch `10/10 parts`
* [x] 3.3. Concise Implementation of Linear Regression `07/07 parts`
* [x] 3.4. Softmax Regression `11/11 parts`
* [x] 3.5. The Image Classification Dataset (Fashion-MNIST) `04/04 parts`
* [x] 3.6. Implementation of Softmax Regression from Scratch `06/06 parts`
* [x] 3.7. Concise Implementation of Softmax Regression `03/03 parts`

## Giai đoạn 2 `(33 parts)`
* [x] 3. Linear Regression Index `01/01 part`
* [x] 3.1. Linear Regression `10/10 parts`
* [x] 3.2. Linear Regression Implementation from Scratch `00/06 parts`
* [x] 3.3. Concise Implementation of Linear Regression `03/05 parts`
* [ ] 3.4. Softmax Regression `00/05 parts`
* [x] 3.5. The Image Classification Dataset (Fashion-MNIST) `00/02 parts`
* [ ] 3.6. Implementation of Softmax Regression from Scratch `00/03 parts`
* [x] 3.7. Concise Implementation of Softmax Regression `00/02 parts`

**Comment by tiepvupsu** [comment]
Cũng phải tầm 100 cái PRs là ít nhỉ.

**Comment by duythanhvn** [comment]
Tầm đó anh ạ.

**Comment by duythanhvn** [comment]
## 3.1. Linear Regression (Giai đoạn 1)
- [x] Phần 1: thanhvinhle26
- [x] Phần 2: thanhvinhle26
- [x] Phần 3: tamnguyenvan
- [x] Phần 4: hungthuanmk
- [x] Phần 5: thduynguyen
- [ ] Phần 6: NhatQuanBui
- [x] Phần 7: thienbuu
- [x] Phần 8: lyphilong
- [x] Phần 9: thunguyenuehk39
- [x] Phần 10: tdh512194
- [x] Phần 11: tmnguyen1403
- [x] Phần 12: honghanhh
- [ ] Phần 13: sudohainguyen
- [x] Phần 14: tamnguyenvan
- [x] Phần 15: tamnguyenvan
- [x] Phần 16: hungthuanmk
- [x] Phần 17: tamnguyenvan
- [ ] Phần 18: NhatQuanBui

**Comment by duythanhvn** [comment]
## 3.2. Linear Regression Implementation from Scratch (Giai đoạn 1)
- [x] Phần 1: lyphilong
- [x] Phần 2: tamnguyenvan
- [ ] Phần 3: longnguyen1984
- [ ] Phần 4: quangnhat185
- [ ] Phần 5: quangnhat185
- [ ] Phần 6: quangnhat185
- [x] Phần 7: thunguyenuehk39
- [x] Phần 8: patrickphatnguyen
- [ ] Phần 9: di-mi-ta
- [x] Phần 10: honghanhh

**Comment by duythanhvn** [comment]
## 3.3. Concise Implementation of Linear Regression (Phase 1)
- [ ] Phần 1: tamnguyenvan
- [x] Phần 2: lyphilong
- [ ] Phần 3: MarkusKhoa
- [ ] Phần 4: tamnguyenvan
- [ ] Phần 5: NhatQuanBui
- [ ] Phần 6: NhatQuanBui
- [ ] Phần 7: longnguyen1984

**Comment by duythanhvn** [comment]
## 3.4. Softmax Regression (Phase 1)
- [ ] Phần 1: honghanhh
- [ ] Phần 2: lyphilong
- [ ] Phần 3: NhatQuanBui
- [ ] Phần 4: lyphilong
- [ ] Phần 5: thunguyenuehk39
- [ ] Phần 6: antran511
- [ ] Phần 7: lyphilong
- [ ] Phần 8: 
- [ ] Phần 9: 
- [ ] Phần 10: 
- [ ] Phần 11:

**Comment by duythanhvn** [comment]
## 3.5. The Image Classification Dataset (Fashion-MNIST) (Phase 1)
- [ ] Phần 1: 
- [ ] Phần 2: 
- [ ] Phần 3: 
- [ ] Phần 4:

**Comment by duythanhvn** [comment]
## 3.6. Implementation of Softmax Regression from Scratch (Phase 1)
- [ ] Phần 1: 
- [ ] Phần 2: 
- [ ] Phần 3: 
- [ ] Phần 4: 
- [ ] Phần 5: 
- [ ] Phần 6:

**Comment by duythanhvn** [comment]
## 3.7. Concise Implementation of Softmax Regression (Phase 1)
- [ ] Phần 1: 
- [ ] Phần 2: 
- [ ] Phần 3:

**Comment by hoangan109** [comment]
mình muốn tham gia hỗ trợ dịch thuật ạ.
